### PR TITLE
feat: Added feature to Create FAQ

### LINKED ISSUE #677: Add option to create FAQ for event
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request
    
- [ ] Chore

**Parent Issue**
#652 

**Description**

Provide an option by which a user can enter question and answer and create FAQ in the app.

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Masquerade0097**
Fixes #677

Changes: 
1. Added option to create FAQ.
2. Refractored the name of FAQRepository to FaqRepository to maintain uniformity throughout the codebase.

Screenshots for the change: 
faqvd

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@iamareebjamal Please review.

File: app/src/main/java/org/fossasia/openevent/app/common/data/network/EventService.java [hunk 1]
```diff
@@ -94,4 +94,7 @@
     @GET("events/{eventId}/event-copyright?include=event&fields[event]=id&page[size]=0")
     Observable<Copyright> getCopyright(@Path("eventId") long eventId);
 
+    @POST("/v1/faqs")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/common/data/network/EventService.java]: Should be just faqs and with the faq get method

File: app/src/main/java/org/fossasia/openevent/app/module/faq/create/CreateFaqPresenter.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @Override
+    public Faq getFaq() {
+        return faq;
+    }
+
+    @Override
+    public void createFaq() {
+
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/faq/create/CreateFaqPresenter.java]: Extra space

**Comment by Masquerade0097** [comment]
@iamareebjamal I've updated the PR, please review.

**Pull request merged**